pixart.js

```diff
@@ -1 +1,12 @@
-
+$("#set-color").on("click", function(event){
+        event.preventDefault()
+        $(".brush").css("background-color", $("#color-field").val())
+});
+for(var i = 0; i < 8000; i++ ){
+    $("body").append("<div class='square'></div>")
+};
+$(".square").each(function(){
+      $(this).on("mouseover", function() {
+          $(this).css("background-color", $("#color-field").val())
+        })
+});
```

Comment:
Oh man I didn't even see this! I guess it's pretty much a bug I didn't see, but I'd prefer feature! I'll think about that one after class